test/unit/specs/components/Personalization/createOnClickHandler.spec.js

```diff
@@ -14,8 +14,11 @@ import createOnClickHandler from "../../../../../src/components/Personalization/
 describe("Personalization::createOnClickHandler", () => {
   let mergeMeta;
   let collectClicks;
-  const event = { type: "blah" };
-  const clickStorage = {};
+  const event = {
+    type: "blah",
```

Comment:
Is `type: "blah"` here serving any purpose?

Reply:
Removed

Reply:
Removed.